tests/pragmas/tuserpragma.nim

```diff
@@ -1,7 +1,19 @@
+discard """
+output: 
+"""
 
 {.pragma: rtl, cdecl, exportc.}
 
 proc myproc(x, y: int): int {.rtl} =
-  nil
+  discard
 
+#####################
 
+# bug #7216
+{.pragma: my_pragma, raises: [].}
+
+doAssert(not compiles(
```

Comment:
This is not really what ``compiles`` is about. The test case should also check the compiler's error message too.

Reply:
Should I put in the separate test file then?

Reply:
Tests can have ``errormsg`` specs.

Reply:
Oh I see what mean. Yes, separate test :-(